backend/experiment/views.py

```diff
@@ -125,12 +125,9 @@ def serialize_experiment(experiment_object, finished=False):
 
 def check_finished_session(experiment, participant):
     ''' Check if there is a finished session for this experiment and participant '''
-    try:
-        Session.objects.get(
-            experiment=experiment, participant=participant, finished_at__isnull=False)
-        return True
-    except:
-        return False
+    count = Session.objects.filter(
+        experiment=experiment, participant=participant, finished_at__isnull=False).count()
+    return count
```

Comment:
As it returns a number now, I think it would be a good idea to update the name of the method as well. Perhaps something like `def get_finished_sessions_count`?